docs-devsite/auth.md

```diff
@@ -1585,6 +1606,8 @@ Promise&lt;void&gt;
 
 Updates the user's phone number.
 
+This method is not available in Node.js.
+
 <b>Signature:</b>
```

Comment:
I'm new to the node.js implementation, but isn't this just updating a field on the user, like updating the password? It doesn't require phone auth to work I'd think. Why isn't this supported?

Reply:
Yes, this requires performing a phone login (which is verified using recaptcha) and obtaining a PhoneAuthCredential which is a parameter to updatePhoneNumber.